src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -430,7 +439,76 @@ public override RunVersionOne VisitRunVersionOne(RunVersionOne v1Run)
                 }
             }
 
-            return null;
+            return run;
+        }
+
+        public static Stack CreateStack(StackVersionOne v1Stack)
+        {
+            Stack stack = null;
+
+            if (v1Stack != null)
+            {
+                stack = new Stack
+                {
+                    Properties = v1Stack.Properties
+                };
+
+                if (!string.IsNullOrWhiteSpace(v1Stack.Message))
+                {
+                    stack.Message = new Message
+                    {
+                        Text = v1Stack.Message
+                    };
+                }
+
+                if (v1Stack.Frames != null)
+                {
+                    stack.Frames = new List<StackFrame>();
+
+                    foreach (StackFrameVersionOne v1StackFrame in v1Stack.Frames)
+                    {
+                        stack.Frames.Add(CreateStackFrame(v1StackFrame));
+                    }
+                }
+
+                if (v1Stack.Tags.Count > 0)
+                {
+                    stack.Tags.UnionWith(v1Stack.Tags);
+                }
```

Comment:
I hadn't caught this before. tags are persisted in the property bag:
```
{
  "properties": {
    "foo": "bar",
    "tags": [ "1", "2" ]
  }
}
```
So I would expect that when you copy the property bag from `v1Stack` to `stack`, the tags come along for the ride, and `stack.Tags` should automatically access them. So is this necessary?

Let's add a test for copying tags, and convince ourselves that it passes even if we remove this code (and similar code in other methods of this class).

Reply:
Don't remember why I did it that way...done.